client/src/tailwind.css

```
@@ -43,13 +51,67 @@
   --color-blue-cool-50: #3a7d95;
   --color-blue-cool-60: #2e6276;
   --color-blue-cool-80: #14333d;
+  --color-blue-cool-90: #0f191c;
 
   --color-gray-100: #fbfcfd;
   --color-gray-200: #f3f3f3;
 
+  --color-gray-cool-10: #dfe1e2;
+  --color-gray-cool-20: #c6cace;
+  --color-gray-cool-30: #a9aeb1;
+  --color-gray-cool-40: #8d9297;
+  --color-gray-cool-50: #71767a;
+  --color-gray-cool-60: #565c65;
+  --color-gray-cool-70: #3d4551;
+  --color-gray-cool-80: #2d2e2f;
+  --color-gray-cool-90: #1c1d1f;
+
   --color-red-300: #e41d3d;
 
+  --color-state-success-dark: #216e1f;
+  --color-state-error-dark: #b50909;
+
   --color-violet-warm-60: #864381;
   --color-violet-warm-70: #5c395a;
   --color-violet-warm-80: #382936;
 }
+
+/**
+  * README:
+  * Overriding USWDS styles here with a mixture of raw CSS and applying
+  * Tailwind classes as needed. This makes having the minimal amount of classes
+  * in the components that leverage the USWDS Table HTML to not require having
+  * a lot of Tailwind class names.
+  */
+
+.usa-table {
+  @apply border-gray-cool-30 w-full rounded-sm border-1;
+  box-shadow: 0px 4px 8px 0px rgba(0, 0, 0, 0.15);
+}
+
+.usa-table--borderless thead th {
+  @apply text-blue-cool-90 bg-gray-cool-10 border-none py-3 uppercase;
+  line-height: 26px;
```

Tailwind allows us to use [custom values ](https://tailwindcss.com/docs/line-height#using-a-custom-value)so we can do something like this for `line-height`:

```suggestion
  @apply text-blue-cool-90 bg-gray-cool-10 border-none py-3 uppercase leading-[26px];
```